src/main.py

```diff
@@ -60,20 +64,7 @@
 If you have any questions, please don't hesitate to get in touch.
 And if you're interested in contributing to our open source project, feel free to join us!
 """
-app = FastAPI(
-    title="Nowcasting API",
-    version=version,
-    description=description,
-    contact={
-        "name": "Open Climate Fix",
-        "url": "https://openclimatefix.org",
-        "email": "info@openclimatefix.org",
-    },
-    license_info={
-        "name": "MIT License",
-        "url": "https://github.com/openclimatefix/nowcasting_api/blob/main/LICENSE",
-    },
-)
+app = FastAPI(docs_url=None, redoc_url=None)
```

Comment:
docs_url could be "/swagger" then I (and maybe ohter people) can still access swagger docs if they want